dsl/pact_test.go

```diff
@@ -176,17 +176,82 @@ func TestPact_Setup(t *testing.T) {
 	if pact.Server == nil {
 		t.Fatalf("Expected server to be created")
 	}
-
-	pact = &Pact{Port: port, LogLevel: "DEBUG"}
-	pact.Setup(false)
-	if pact.Server != nil {
+	defer pact.Teardown()
```

Comment:
on my local machine i was getting hanging mock servers so i just added code to cleanup. I get it will break if teardown() breaks.